footprints/main/views.py

```diff
@@ -534,3 +543,44 @@ def form_valid(self, form):
         send_mail(subject, tmpl.render(Context(form_data)), sender, recipients)
 
         return super(ContactUsView, self).form_valid(form)
+
+
+class SignS3View(LoggedInMixin, View):
```

Comment:
OK, third time's the charm. I'm tasking myself with pulling the `SignS3View` out into its own app so we don't end up with it proliferating across all our projects.